cookbooks/2d_annulus_visualization/doc/2d_annulus_visualization.md

```diff
@@ -0,0 +1,115 @@
+
+The input file to generate the model that created the accompanying figures can be found at[cookbooks/twoD_annulus_visualization/2D_annulus_example.prm]. The ASPECT parameter file is a simple modification of the onset_of_convection cookbook contributed by Juliane Dannberg.
+
+## How to get started
+Since this Python script utilizes pyvista, you should install this package using anaconda following the instructions in the README.md file found in the directory [/aspect/contrib/python/]. Other python scripts that contribute to ASPECT's  functionality can be found within that directory, as well as another 2D pyvista visualization tutorial for a box geometry called 'pyvista_2d_example.py', contributed by John Naliboff.
+
+More information about installing Python via anaconda can be found here: https://docs.anaconda.com/free/anaconda/install/index.html
+
+For those without a lot of python scripting experience, copy this script ".py" file into some directory where you store analysis scripts or make a new directory and store the script there. Make sure the "dir" variable within the script is the full path to the model directory in which "output/" is a subdirectory containing the files log.txt, solution.pvd, and  the "solution/" directory. Follow instructions in [/aspect/contrib/python/README.md] to set up your anaconda environment with pyvista. Then, in a terminal window and within the analysis script directory containing the ".py" file, run
+
+    $ python pyvista_2d_annulus_example.py
+
+
+## Modifying inputs to the scripts
+
+Most of the script can be run without modification for your models. The only inputs you will need to adjust yourself are:
+
+    dir = 'String/Path/To/Your/Model/Directory/'
+
+where the model directory is the one which contains the [output/] directory.  
+
+The initial and final integer number of solution outputs you would like to visualize:
+
+    out_n_min=0 # The initial solution file to visualize: i.e. 'solution-00000.pvtu'
+    out_n_max=25 # The final solution file to visualize: i.e. 'solution-00025.pvtu'
+
+and the inner and outer radius of the annulus from the geometry of your model:
+    r_inner = 3481000 # units in m
+    r_outer = 6371000 # units in m.
+
+## What the script does
+
+This script can be treated a template and modified based on the fields you are interested in examining. A simple pyvista plot to show and not save the image has the basic form:
+
+    pl = pv.Plotter()
+    pl.add_mesh(
+        mesh,
+        scalars="p"
+        clim=[0,1e11], # color bar range from 0 to 100 GPa
+        cmap="RdBu",   # color map to use
```

Comment:
may be after this you could add a sentence or two about colormaps. 
see https://www.fabiocrameri.ch/colourmaps/
and this article why it matters:
@article{crsh20,
author={Crameri, Fabio and Shephard, Grace E. and Heron, Philip J.},
title={The misuse of colour in science communication},
journal={Nature Communications},
pages={5444},
volume={11},
doi={10.1038/s41467-020-19160-7},
year={2020}}

Reply:
Do you recommend I change RdBu to Crameri's vik? I usually use Crameri color maps, however I neglected to add it to my script (perhaps to avoid another python package for a user to install, no real reason)?

Reply:
not necessary, with the new paragraph you added about colormaps.